src/nuclio/functions/version/version-triggers/version-triggers.component.js

```diff
@@ -409,7 +409,8 @@
          */
         function updateTriggerList() {
             ctrl.triggers = lodash.map(ctrl.version.spec.triggers, function (trigger) {
-                var triggersItem = lodash.assign(lodash.cloneDeep(trigger), createTriggerItem(trigger));
```

Comment:
Explanation for future developers:
If the user imports from YAML a function that already has Cron triggers with, then the call to `createTriggerItem` here sends the trigger object itself, and not a clone. Then in `createTriggerItem` there is this line:
```js
var triggerItem = lodash.assign({}, noTriggerProvided ? {} : trigger, { … });
```
where `trigger` is a reference to the same object.
This means that `triggerItem` has `attributes` property that points to the exact same `attributes` property of `trigger`, which is the original object that came from YAML.
Then, the above `triggerItem` is returned from `createTriggerItem` into the `triggersItem` variable here, which later gets its `attributes.schedule` reduced from 6 cron pattern fields to 5 (e.g. `0 0 5 * * *` to `0 5 * * *`).
Since this is referencing the same `attributes` object of the _original_ trigger object — when the user just deploys the function without editing this trigger, then the request payload has 5 fields for cron and not 6.

The solution here is to pass a _clone_ of the `trigger` parameter into `createTriggerItem`, rather than the original object. Then the original object has 6 fields unchanged by the following logic here. Only the clone gets its 6 fields reduced to 5 and displayed in the view.